code/datums/elements/on_fire_connector.dm

```diff
@@ -0,0 +1,48 @@
+///simple element to handle: 1. mobs that are on fire bumping into other mobs
+///and 2. mobs on fire moving into a turf containing mobs not on fire, but not bumping into them
+/datum/element/on_fire_connector
```

Comment:
@LemonInTheDark  one of the two connect_loc registers to normal human movement. it was the less ugly of the two to move away from connect_loc, so i did this one.